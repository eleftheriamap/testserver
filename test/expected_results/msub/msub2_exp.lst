
test/exp/test/gen/msub/msub2_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	11209294 	add	w20, w20, #0x824
   4:	5123939c 	sub	w28, w28, #0x8e4
   8:	d158f463 	sub	x3, x3, #0x63d, lsl #12
   c:	1b1c8e97 	msub	w23, w20, w28, w3
  10:	8a000000 	and	x0, x0, x0
