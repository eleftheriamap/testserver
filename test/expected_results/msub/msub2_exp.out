Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = ffffffffff9c3000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000824
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 00000000ffe49010
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 00000000fffff71c
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x11209294
0x00000004 : 0x5123939c
0x00000008 : 0xd158f463
0x0000000c : 0x1b1c8e97
0x00000010 : 0x8a000000
